app/helpers/fact_values_helper.rb

```diff
@@ -60,7 +60,7 @@ def fact_breadcrumbs
           url: (fact_values_path if authorized_for(hash_for_fact_values_path)),
         },
         {
-          caption: Setting[:display_fqdn_for_hosts] ? params[:host_id].split('.')[0] : params[:host_id],
+          caption: Setting[:display_fqdn_for_hosts] ? params[:host_id] : params[:host_id].split('.')[0],
```

Comment:
Instead of having multiple sections in which we need to differentiate if fqdn or hostname should be displayed, I would suggest to have helper method for it which can be used in this sections. 

This method can then have a quick unit test, too.

Reply:
Yeah that's a good idea. Will do.